R/maraca.R

```diff
@@ -52,7 +53,7 @@ maraca <- function(
       arm = "arm",
       value = "value"
     ),
-    fixed_followup_days = NULL,
```

Comment:
considering that now this package is on CRAN, it's generally bad practice now to remove a feature like this. People will get angry because now their scripts don't work anymore.

I am not expecting a horde to scream at us though. Just something that one needs to consider in the future.

Reply:
I agree but I think it's good that we changed this now at least. At the moment, it's maybe not strictly necessary to have this compulsory but for some of the future development plans we have, it will be needed.